### PR TITLE
Add app_launcher_visible to Access Application

### PR DESCRIPTION
**Pull request by AkemiDavisson**
Requires a new cloudflare-go version with https://github.com/cloudflare/cloudflare-go/pull/739 before this can be merged

### CODE REVIEW

**Comment by AkemiDavisson** [comment]
@jacobbednarz can we get this merged/released as well. Or are we waiting on the cloudflare/go release. Thanks!

**Comment by jacobbednarz** [comment]
the latter; I've labelled this as such last week to track the progress.

**Comment by AkemiDavisson** [comment]
Ah missed the label. Awesome. Thanks!

File: website/docs/r/access_application.html.markdown [hunk 1]
```diff
@@ -46,54 +46,54 @@ The following arguments are supported:
 
 -> **Note:** It's required that an `account_id` or `zone_id` is provided and in most cases using either is fine. However, if you're using a scoped access token, you must provide the argument that matches the token's scope. For example, an access token that is scoped to the "example.com" zone needs to use the `zone_id` argument.
 
-* `account_id` - (Optional) The account to which the access application should be added. Conflicts with `zone_id`.
```
Comment (jacobbednarz) [inline website/docs/r/access_application.html.markdown:49]: please revert the markdown list back to `*`. we use it in all other documentation and will also be the automatically generated version once it comes into this repository.

**Pull request merged**